vello_tests/src/snapshot.rs

```diff
@@ -95,33 +106,78 @@ impl Snapshot<'_> {
     }
 }
 
+/// The directory to store a snapshot test within.
+#[derive(Clone, Copy)]
+pub enum SnapshotDirectory {
+    /// Run in a smoke test directory.
+    ///
+    /// Snapshots in this directory should be small, because they commit binary data directly to the repository.
+    /// This test will ensure that files produced are no more than 4KiB.
+    Smoke,
+    /// Run the test in a git LFS managed directory.
+    ///
+    /// This test will ensure that files produced are no larger than 128KiB.
+    Lfs,
+}
+
+impl SnapshotDirectory {
+    fn max_size_in_bytes(self) -> u64 {
```

Comment:
Not that it does matter, I think this could even be `const`

Reply:
It's always used with dynamic arguments, but I see your point. As it happens, I originally wrote it as `const`, then removed that again. If you add this as a suggestion, I'll apply it.

Reply:
As I say it doesn't really matter, it's not that it helps much, I think you could keep it as that, just something I noticed